docker/all-in-one/.env

```diff
@@ -1,6 +1,6 @@
 # See the README.md file for information on how to generate the JWT_SECRET and APP_KEY
-APP_KEY=
-JWT_SECRET=
+APP_KEY=base64:dx9+S5nIAyKz3IV39JbdlNhl3lNMyQI4jDHuSNo1M5E=
+JWT_SECRET=Fqr95S7sOQ3szWlOgAwe+RVN4BZ9XhFLc720JGqelkM=
```

Comment:
I assume you meant to open this against your own fork. If you're using these keys in production you should rotate them ASAP.